cfme/tests/generic_objects/test_instances.py

```diff
@@ -212,6 +199,7 @@ def test_generic_objects_with_buttons_ui(appliance, request, definition, service
             assert view.toolbar.button(generic_button.name).custom_button.is_displayed
 
 
+@pytest.mark.meta(blockers=[BZ(1648243, forced_streams=['5.9'])])
```

Comment:
Q: @digitronik I think devs will keep only one BZ for this, it will be either this or 1648658. Should we put both or wait till one will be closed as a duplicate?

Reply:
@lina-nikiforova awesome they duplicated one BZ... but I don't understand on what basis they duplicated `1648243` as its logged on 09/11 and `1648658` logged on `11/11` :). Mostly they choose first come bug and duplicate other; on that basis only I choose this...